architecture/service_catalog/index.adoc

```diff
@@ -101,6 +101,24 @@ references to the binding secret from the deployment. Otherwise, the next
 rollout will fail.
 ====
 
+[[service-catalog-route-planning]]
+== Service catalog compatibility with services outside of the cluster
+
+As of {product-title} version 3.10, a user can specify a Kubernetes service
+outside of the cluster to be consumed by other users using firewall rules. This
+is up to the user to ensure that the service is able to be consumed by adding
+rules. {product-title} ensures that the network isolation is automatically
+allowing connectivity to occur.
+
+Administrators can use `NetworkPolocy` objects to declare appropriate access to
+these resources from a user's project or namespace. For example, from the
+Ansible broker or the Template broker. These `NetworkPolocy` objects are
+triggered by using the service catalog through provisioning or binding.
+
+See the <here> for more information on how to trigger the service catalog.
+
+<We need to document how to make it possible to access the provisioner from projects (like we do with the router). Then we need to document that the task that is provisioning the service set up appropriate network policy rules to allow access.>
```

Comment:
I'm not sure how thorough we need to be... I think there are three cases we could enumerate:

1) The SC installs things inside the project -- Needs NetPol rules to allow access in the project
2) The SC installs things in a different namespace in the cluster -- The target may need NetPol rules to allow the source project to talk to it
3) The SC enables access to an out-of-cluster service -- Nothing in the cluster needs to be changed.